packages/patternfly-4/react-core/src/components/Text/TextList.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+
+export enum TextListVariants {
+  ul = 'ul',
+  ol = 'ol',
+  dl = 'dl'
+}
+
+export interface TextListProps extends React.HTMLProps<HTMLElement> {
+  /** Content rendered within the TextList */
+  children?: React.ReactNode;
+  /** Additional classes added to the TextList */
+  className?: string;
+  /** The text list component */
+  component?: TextListVariants;
```

Comment:
Please change to `TextListVariants | 'ul' | 'ol' | 'dl'`

Reply:
same comment as above.